src/ontology/uberon.Makefile

```diff
@@ -98,250 +120,43 @@ $(OWLSRC): $(SRC) $(COMPONENTSDIR)/disjoint_union_over.ofn $(REPORTDIR)/$(SRC)-g
-
-# including the imports would lead to circularity, so we remove these here
-#$(COMPONENTSDIR)/phenoscape-ext.owl: $(TMPDIR)/phenoscape-ext-src.owl tmp/core.owl
-#	owltools $(UCAT) $< --remove-imports-declarations -o -f functional $@
-
-# Not used anywhere so commenting out
-#$(TMPDIR)/phenoscape_homology.owl:
-#	wget --no-check-certificate https://raw.githubusercontent.com/phenoscape/phenoscape-ontologies/master/demo/phenoscape_homology.owl -O $@
-
-## MERGED UNREASONED ONTOLOGY
-##
-## TODO - restore Disjoints. _ OLD NOTE.  DELETE?
-## TODO - get rid of declarations and inferred subclass axioms for other ontology classes _OLD NOTE. DELETE?
-
-# TODO - Make uberon-bridge-to-bfo.owl a component.  BUT it needs to stay in the bridges directory!
-$(TMPDIR)/unreasoned.owl: $(OWLSRC) $(BRIDGEDIR)/uberon-bridge-to-bfo.owl # $(COMPONENTSDIR)/phenoscape-ext.owl
-	$(OWLTOOLS) $^ --merge-support-ontologies -o -f functional $@
-
-# First pass at making base module
-# Currently this will be missing the temporary reflexivity axioms.
-# should this include downward-injected axioms, e.g on ZFA?
-#uberon-base.owl: $(TMPDIR)/unreasoned.owl
-#	$(OWLTOOLS) $< --remove-imports-declarations --remove-axioms -t Declaration --set-ontology-id -v $(RELEASE)/$@ $(URIBASE)/uberon/$@ -o $@.tmp && mv $@.tmp $@
-
-# TODO document collected-* pattern (import files see http://uberon.github.io/downloads.html#multiont)
-# ----------------------------------------
-# STEP 3: Perform reasoning and create release ext.owl file
-# ----------------------------------------
-
-# ext.owl is the release file that includes full imports and inter-ontology axioms
-#ext.owl: $(TMPDIR)/materialized.owl
-#	$(ROBOT) reason -i $< -r elk relax reduce -r elk annotate --ontology-iri $(URIBASE)/uberon/$@ -V  $(RELEASE)/$@ -o $@
-
-## TESTING - NEW
-$(TMPDIR)/is_ok: $(TMPDIR)/materialized.owl
-	$(OWLTOOLS) $< --run-reasoner -r elk -u > $@.tmp && mv $@.tmp $@
-
-# somewhat awkward: we temporarily inject reflexivity axioms
-# **Hacking_Feb_2022** Notes start here
-# Background: 
-## This ends up in the release file, reflexivity_axioms.owl temp merge as consistency QC
-## This is only a QC.  Rationale - this would catch some errors because expands part disjointness ()
-## **Hacking_Feb_2022** TODO - rewrite as as expansions from annotation axioms using SPARQL. Expanded axioms --> component (as for taxon restrictions).
-
-TMP_REFL=$(COMPONENTSDIR)/reflexivity_axioms.owl
+# Step 2: Reasoning.
+# For the temporary injection of property chains, see
+# <https://github.com/obophenotype/uberon/issues/2381>
 DEVELOPS_FROM_CHAIN=$(COMPONENTSDIR)/develops-from-chains.owl
-# see https://github.com/obophenotype/uberon/issues/2381
-
-$(TMPDIR)/materialized.owl: $(TMPDIR)/unreasoned.owl $(TMP_REFL)
-	$(ROBOT) merge -i $< -i $(DEVELOPS_FROM_CHAIN) --collapse-import-closure false \
```

Comment:
Given the `--collapse-import-closure false` in the makefile goal, it seems to me that the intention was that ext.owl (immediately dependent on materialized.owl) is supposed to be uberon.owl _but with imports merged in_. However, downloading ext.owl it seems that merging was indeed happening somewhere upstream, like `unreasoned.owl`.

Reply:
Imports are merged right at the beginning of the (original) pipeline, during the “preprocessing” step:

```Makefile
$(OWLSRC): $(SRC) $(COMPONENTSDIR)/disjoint_union_over.ofn $(REPORTDIR)/$(SRC)-gocheck $(REPORTDIR)/$(SRC)-ic>
        @echo "STRONG WARNING: issues/contributor.owl needs to be manually updated."
        $(ROBOT) merge -i $< \
                        -i $(COMPONENTSDIR)/disjoint_union_over.ofn \
                        -i issues/contributor.owl \
                expand --no-expand-term http://purl.obolibrary.org/obo/RO_0002175 \
                        -o $@
```

Now, it’s entirely possible that this was not intentional, and only caused by the fact that whoever wrote this didn’t know that `robot merge` merges the import **by default**, unless `--collapse-import-closure` is explicitly set to _false_.

But whether this was the intention or not, I wrote this PR with the aim of _not changing the current behaviour_. If the current behaviour was not actually wanted, that can be fixed in a later PR ­– where it would actually be much easier to fix with a readable, maintainable Makefile!

Reply:
> whoever wrote this didn’t know that robot merge merges the import by default, unless --collapse-import-closure is explicitly set to false.

To be fair to that person: ROBOT’s inline help is probably at fault here:
```sh
$ robot merge --help
usage: robot merge --input <file> --input <file> --output <file>
[…]
 -c,--collapse-import-closure <arg>   if true, the imports closure will be
                                      merged
```
With such a phrasing, I think anyone reading this would conclude that the default value for this option is **false**, _i.e._ that the imports closure is _not_ merged by default.

One needs to consult the [online help](http://robot.obolibrary.org/merge) to learn that it is the opposite.

Reply:
Lets roll with this, as it is the status quo!